src/Views/Admin/Pages/templates/setup-page/index.html.php

```diff
@@ -0,0 +1,114 @@
+<div class="wrap" class="give-setup-page">
+
+	<h1 class="wp-heading-inline">
+		<?php echo __( 'Setup GiveWP', 'give' ); ?>
+	</h1>
+
+	<hr class="wp-header-end">
+
+	<!-- Configuration -->
+	<?php
+		echo $this->render_template(
+			'setup-page/section',
+			[
+				'title'    => 'Create your first donation form in minutes',
+				'badge'    => '<span class="badge badge-review">5-10 Minutes</span>',
+				'contents' => $this->render_template(
+					'setup-page/row-item',
+					[
+						'class'       => 'configuration',
+						'icon'        => $this->image( 'configuration@2x.min.png' ),
+						'icon_alt'    => esc_html__( 'First-Time Configuration', 'give' ),
+						'title'       => esc_html__( 'First-Time Configuration', 'give' ),
+						'description' => esc_html__( 'Every fundraising campaign begins with a donation form. Click here to create your first donation form in minutes. Once created you can use it anywhere on your website.', 'give' ),
+						'action'      => '<a href="' . admin_url( '?page=give-onboarding-wizard' ) . '"><span class="screen-reader-text">Configure GiveWP</span><i class="fas fa-chevron-right"></i></a>',
```

Comment:
This implementation creates a slightly awkward experience for keyboard users. When you tab through the page, the arrow is enclosed in a box, using the default focus state. While this is a workable solution, revisiting the styles could help to ensure that box shadow that appears to help highlight the link is consistent with focus states used for buttons, and uses a display setting that ensures the box matches the arrow's width. See this screenshot for current behavior:

<img width="1574" alt="Screen Shot 2020-07-16 at 2 10 14 PM" src="https://user-images.githubusercontent.com/5186078/87723359-90ecf600-c76e-11ea-944f-c89697c09448.png">